.github/workflows/release.yml

```diff
@@ -50,15 +50,15 @@ jobs:
 
 
   wheels:
-    if: ${{ github.event_name == 'workflow_dispatch' || startsWith(github.ref, 'refs/tags/v') }}
+    if: ${{ github.event_name == 'workflow_dispatch' || startsWith(github.ref, 'refs/tags/tpypi') }}
     name: ${{ matrix.py }} ${{ matrix.os }} ${{ matrix.arch }}
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
         arch: [native, arm64]
-        py: [cp38, cp39, cp310, cp311]
+        py: [cp39, cp310, cp311, cp312]
```

Comment:
There are no binaries for numpy 1.23 and Python 3.12, and this case is not in the PR tests. So it’s likely to fail. To get this PR merged I’d recommend to avoid building wheels on 3.12 for now, or build those with np1.26

Reply:
Ok, I will change it in #164